jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/Config.java

```diff
@@ -90,7 +91,7 @@ private Config() {
    * @return A HashMap of the values in the section
    */
   public static Map<String, String> getSection(String section) {
-    HashMap<String, String> sectionValues = new HashMap();
+    HashMap<String, String> sectionValues = new HashMap<String, String>();
```

Comment:
`<String,String>` can be replaced with a `<>` operator.

Reply:
Done